config/helmet.js

```diff
@@ -18,6 +18,12 @@ module.exports = {
         "'self'",
         'www.google-analytics.com',
       ],
+      'frame-ancestors': [
```

Comment:
I don't think we need `frame-ancestors`. The finding is that either `frame-ancestors` OR `form-action` is missing. But we seem to have `frame-ancestors` covered by the defaults (per https://helmetjs.github.io/) and scan result looks ok for the pages that _have_ included helmet:

<img width="689" alt="image" src="https://user-images.githubusercontent.com/1211146/125995107-b3a3d63c-0551-471b-bb1e-f42efce82279.png">

But I don't know why some other number of pages don't seem to picked up any of this helmet config (e.g. https://federalistapp-staging.18f.gov/images/logos).

Reply:
@pburkholder it's not a big to narrow `frame-ancestors` down, we don't want to IFramed so changing it to `'none'` is fine.